src/backend/distributed/commands/cascade_table_operation_for_connected_relations.c

```diff
@@ -326,6 +326,11 @@ GetCascadeOperationFunction(CascadeOperationType cascadeOperationType)
 			return UndistributeTable;
 		}
 
```

Comment:
Self TO-DO:
- [ ] Rename cascade -> cascade_via_foreign_keys
- [ ] Fix errors thrown for foreign keys in CreateCitusLocalTable, give hint for cascade option
- [ ] Add a test for fkeys between partitioned tables
- [ ] Add simple mx tests
- [ ] Review `create_citus_local_table_cascade tests`